challenge_7/python/wost/find_missing_number.py

```diff
@@ -0,0 +1,14 @@
+def find_biggest(a):
+	biggest = a[0]
+	for i in a:
+		if i >= biggest:
+			biggest = i
+	return biggest
+
+def find_missing_number(c):
+	b = find_biggest(c)
```

Comment:
will this work if your range does not start at zero? For example, what if the range was `[9, 7, 5, 8]` would your solution return 6?